Dockerfile

```diff
@@ -24,6 +25,8 @@ RUN env
 RUN python3 -m squad.frontend
 RUN ./manage.py collectstatic --noinput
 
+RUN chown -R www-data /app
```

Comment:
why do you need this? having source tree be read-only for the user that runs the service is better for security

Reply:
It's because I was using it as a dev environment, but there's probably a better way. I'll update the PR